### PR TITLE
Resolve relative $refs, bump up parser version

### LINKED ISSUE #30: Import document with $ref not able to resolve
#### Describe the bug
When trying to import a document with references, those references are not resolved against the original document. For example using the example:
https://raw.githubusercontent.com/asyncapi/avro-schema-parser/master/tests/asyncapi-avro-1.9.0.yaml
It is unable to resolve the `schemas/Person-1.9.0.avsc` and looks locally on the system.

I have also tried putting all the async.yaml and related artifacts onto file system of the app however the app is not able to resolve them. Similarly placing them into `public/` folder resolves references at the wrong spot instead looking at `/usr/src/app` instead of `/usr/src/app/public` 

#### How to Reproduce
Steps to reproduce the issue. Attach all resources that can help us understand the issue:
Screenshot_2020-08-26 AsyncAPI 2 0 0 playground

Scenario 1 HTTP

1. Go to https://playground.asyncapi.io/?embed=1
2. Put `https://raw.githubusercontent.com/asyncapi/avro-schema-parser/master/tests/asyncapi-avro-1.9.0.yaml` click import - error appears

Scenario 2 local file

1. Clone playground repo
2. Clone https://github.com/asyncapi/avro-schema-parser/tree/master/tests into public folder
3. Perform docker build and run
4. Import http://$DOCKERHOST:83/asyncapi-avro-1.9.0.yaml - error appears

#### Expected behavior

1. $ref should be resolved relative to original resource if possible - by client if required
2. `file:///` handling should be clarified so that something like a volume mount can work, any chance of using local files?

### PR DESCRIPTION
**Pull request by michaelpro1**
**Description**
Supports resolving relative references by passing in the `parserOptions.path` in the request. 
Note that this change will need to be made to the react view once that is added.
It has been tested with the https://raw.githubusercontent.com/asyncapi/avro-schema-parser/master/tests/asyncapi-avro-1.9.0.yaml which resolves it correctly, when the relative ref is not resolvable it prints out the 404.

**Related issue(s)**
Fixes #30

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -15,7 +15,7 @@
     "@asyncapi/generator": "^1.0.0-rc.10",
     "@asyncapi/html-template": "^0.9.0",
     "@asyncapi/markdown-template": "^0.9.0",
-    "@asyncapi/parser": "^0.28.1",
+    "@asyncapi/parser": "^1.0.0-rc.3",
```
Comment (derberg) [inline package.json:18]: please use latest rc.4

**Review by derberg — COMMENTED** [review]
LGTM but I've never pushed to Playground before so @fmvilas also have to check to approve it.

File: package.json (hunk 1 above)
Comment (michaelpro1) [reply package.json]: Done. I also wasn't sure but do I need to check in the `package-lock.json` or omit it from PR?

Comment (fmvilas) [reply package.json]: My recommendation is that you leave package-lock.json out of the PR. We'll handle it ourselves.

Comment (michaelpro1) [reply package.json]: Done, I've cherry-picked the commits that didnt include that change

**Comment by germanschnyder** [comment]
Just curious, I thought this was already working because of https://github.com/asyncapi/playground/pull/16, but apparently is not ?

**Comment by derberg** [comment]
@germanschnyder good catch. Looks like it was lost during refactoring, have a look at [this](https://github.com/asyncapi/playground/commit/5008038d3c805ae5829c887527be9d8b003ee40b#diff-bba7be6fd99c4dd55f120d8af6e36d14L26) and [that](https://github.com/asyncapi/playground/commit/5008038d3c805ae5829c887527be9d8b003ee40b#diff-fc4c1da0491c3b14a4abcb245fa507a0L16)

@fmvilas I'm leaving merge, package-lock update, and deploy to you 😉

**Comment by fmvilas** [comment]
Merged and deployed :)

Thanks a lot, @michaelpro1!

**Pull request merged**